frontend/src/metabase/admin/upsells/UpsellWhitelabel.tsx

```diff
@@ -0,0 +1,36 @@
+import { t } from "ttag";
+
+import { useHasTokenFeature } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+
+import { UpsellBigCard } from "./components";
+import S from "./components/Upsells.module.css";
+import { useUpsellLink } from "./components/use-upsell-link";
+
+export const UpsellWhitelabel = ({ source }: { source: string }) => {
+  const isWhitelabeled = useHasTokenFeature("whitelabel");
+  const docsLink =
+    "https://www.metabase.com/docs/latest/configuring-metabase/appearance";
```

Comment:
Is there a hook for looking up docs links?

Reply:
There is this hook, but I didn't use it because the UTM params there are optional + .
https://github.com/metabase/metabase/blob/master/frontend/src/metabase/common/hooks/use-docs-url/use-docs-url.ts

Furthermore, @trinya said that the secondary docs links should have the exact same UTM params as the primary upsell link so I thought the existing upsell hook might be handy for that.

Maybe I can use `getDocsUrl` in order to get the version dynamically, but then pass it to the upsell hook in order to match UTM params?

Let me try that if you don't have strong objections.

Reply:
How does this look, @rafpaf ?

https://github.com/metabase/metabase/pull/53566/commits/6cbeb2a112d5890ebd5420ba2c2197cf1c59ef4b
https://github.com/metabase/metabase/pull/53566/commits/d375b4a4904db0bf51ce8d71ab8d9e6e35c62c66